Cinebase/Core/Formatters/DateFormatter.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  DateFormatter.swift
+//  Cinebase
+//
+//  Created by Gustavo Villar on 5/16/17.
+//  Copyright © 2017 Globant. All rights reserved.
+//
+
+import Foundation
+
+class DateFormatter {
+    
+    enum Style {
+        case date
+        case dateTime
+    }
+    
+    private let formatter = Foundation.DateFormatter()
+    
+    func format(_ date: Date, style: Style) -> String {
+        switch style {
```

Comment:
yes 
var dateStyle
var timeStyle

each one with the switch, what do you think? i think it will be better to add and modify new formatter styles, what do you think?